### PR TITLE
Enable tests for all platforms

### PR DESCRIPTION
**Pull request by dmcgowan**
Try to run tests on more platforms

### CODE REVIEW

File: fs/copy_unix.go [hunk 1]
```diff
@@ -71,6 +71,10 @@ func copyFileContent(dst, src *os.File) error {
 func copyXAttrs(dst, src string, excludes map[string]struct{}, errorHandler XAttrErrorHandler) error {
 	xattrKeys, err := sysx.LListxattr(src)
 	if err != nil {
+		if os.IsPermission(err) {
```
Comment (samuelkarp) [inline fs/copy_unix.go:74]:
```suggestion
		if os.IsPermission(err) && runtime.GOOS == "darwin" {
```
?

**Review by samuelkarp — COMMENTED** [review]
I've been meaning to do FreeBSD tests for this too on Cirrus, though that can be a separate PR.

**Comment by samuelkarp** [comment]
It'd also be good to test more filesystems: https://github.com/containerd/continuity/issues/172

**Comment by dmcgowan** [comment]
@samuelkarp agreed, would be a little bit more involved. I'm not completely sure about the Windows test, at least what has changed, we should at least have the tests pass for our CI environment. Expanding to more environments would be good though given how the fs library is used.

**Review by samuelkarp — APPROVED** [review]

**Comment by samuelkarp** [comment]
I opened https://github.com/containerd/continuity/issues/221 to track adding coverage for more filesystems.

**Review by estesp — APPROVED** [review]

**Pull request merged**